src/org/wof/foodministry/FoodEntry.java

```diff
@@ -28,5 +28,29 @@ public String getDay() {
     
     public void setDay ( String day ) {
         this.day = day;
-    } 
+    }
+    public int getweight() {
+	return weight;
+    }
+    public void setweight ( int weight ) {
+        this.weight = weight;
+    }
+    public int getDate () {
+	return date;
+    }
+    public void setDate ( int date ) {
+        this.date = date;
+    }
+    public String getTime() {
```

Comment:
Pay special attention to the name of your class variables.
ie. _Time vs time_ and and _Location vs. location_

If you create a new variable that is not part of your class, what do you expect to happen when you try to use those get and set methods?